projects/go-ethereum/project.yaml

```diff
@@ -0,0 +1,12 @@
+homepage: "https://github.com/ethereum/go-ethereum"
+primary_contact: "Adam@adalogics.com"
```

Comment:
Please add the ethereum dev as primary here, also, your email is repeated below, so move the official dev address here.

Reply:
Fixed with https://github.com/google/oss-fuzz/pull/4515/commits/9cfcc1b52e2862b7f5871bc04d00577d5d518211